### PR TITLE
Update steps to upload file for location (set-up-call-analytics.md)

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/8369.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/set-up-call-analytics.md [hunk 1]
```diff
@@ -50,12 +50,12 @@ If you're a Teams or Skype for Business admin, you can use an existing tenant an
 
 - To download an existing data file, go to **Microsoft Teams admin center** > **Analytics & reports** > **Call Quality Dashboard** > **Upload now**. In the **My uploads** list, click **Download** next to the file you want. 
 
-- To upload the new file, go to **Microsoft Teams admin center** > **Locations**, and then select **Upload location data** or **Replace location data**.
+- To upload the new file, see [Add and update reporting labels](https://docs.microsoft.com/microsoftteams/learn-more-about-site-upload).
```
Comment (JohanFreelancer9) [inline Teams/set-up-call-analytics.md:53]:
```suggestion
- To upload the new file, see [Add and update reporting labels](/microsoftteams/learn-more-about-site-upload).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@SerdarSoysal PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by tiburd** [comment]
reviewed + confirmed

**Pull request merged**